test/phabricator_test/mock_phabricator.py

```diff
@@ -0,0 +1,340 @@
+import datetime
+from reviewrot.basereview import LastComment
+
+class MockPhabricator:
+    def __init__(self):
+        return
+
+    def mock_phabricator_differential_query(status, responsibleUsers, phab):
+
+        res = [
+            {
+                'reviewers': [
+                    'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                    'PHID-USER-xxxxxxxxxxxxxxxxxxxx'
+                ],
+                'lineCount': '2',
+                'repositoryPHID': None,
+                'id': 1706,
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'title': 'Title 1',
+                'activeDiffPHID': 'PHID-DIFF-xxxxxxxxxxxxxxxxxxxx',
+                'branch': 'new_input_data',
+                'dateModified': '1553524722',
+                'status': '2',
+                'testPlan': '',
+                'commits': [],
+                'dateCreated': '1553065630',
+                'hashes': [],
+                'properties': [],
+                'diffs': [
+                    '3605',
+                    '3599'
+                ],
+                'phid': 'PHID-DREV-xxxxxxxxxxxxxxxxxxxx',
+                'uri': 'dummy.url',
+                'css': [],
+                'summary': 'This is a summary.',
+                'statusName': 'Accepted'
+            },
+        ]
+        return res
+
+    def mock_phabricator_get_comments_single(self):
+        return [
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            }
+        ]
+
+    def mock_phabricator_get_comments_many(self):
+        return [
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxx3',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            },
+            {
+                'action': 'comment',
+                'authorPHID': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                'revisionID': '1',
+                'content': 'This is some content',
+                'dateCreated': '1551763640'
+            }
+        ]
+
+    def mock_phabricator_get_revision_comments(self):
+        return [[
```

Comment:
Because the function call will return a list of dictionary objects. See [here](https://secure.phabricator.com/conduit/method/differential.getrevisioncomments/)

Reply:
Can we resolve the conversation here?